Source/Classes/View/CardView/Front/SmallFrontView.swift

```diff
@@ -1,82 +1,283 @@
-import UIKit
-
-class SmallFrontView: CardView {
+//
+//  SmallFrontView.swift
+//  MLCardDrawer
+//
+//  Created by Jonathan Scaramal on 30/06/2021.
+//
 
-    @IBOutlet weak var number: CardLabel!
+import UIKit
 
-    @IBOutlet weak var debitImage: UIImageView!
+public class SmallFrontView: CardView {
+    @IBOutlet weak var expirationDate: CardLabel!
+    @IBOutlet weak var name: CardLabel!
     @IBOutlet weak var paymentMethodImage: UIImageView!
+    @IBOutlet weak var bankImage: UIImageView!
+    @IBOutlet weak var number: CardLabel!
+    @IBOutlet weak var securityCodeCircle: CircleView!
+    
+    @IBOutlet weak var safeZone: UIView!
+    
+    // Constraints
+    @IBOutlet weak var numberToNameLeadingConstraint: NSLayoutConstraint!
+    var numberToNameLeadingProgConstraint : NSLayoutConstraint?
```

Comment:
### Advice

Group all `@IBOutlet weak var` and then the vars, to orginize the code

Reply:
They're intentionally ordered this way because one is created with IB, the other programmatically but both refers to the same pair of Views.